README.md

```diff
@@ -1,62 +1,97 @@
-1. Patch the changelog file:
-    ```bash
-   ./gradlew patchChangelog
-    ```
-1. Update the version in `gradle.properties` to include a SNAPSHOT version, e.g. change `pluginVersion=0.1.0` to `pluginVersion=0.2.0-SNAPSHOT`.
-
-## Installation
-After building the plugin you can drag & drop the file `./build/distributions/intellij-appmap.zip` onto the main window of IntelliJ to install the plugin. A restart of your IDE is required to use the plugin.
-
-## Usage
-A custom file editor is opened when a `.appmap.json` file is opened.
-If JCEF is unsupported on your system, then an error message will be shown instead.
-
-## Releasing
-- Find the latest official release from [GitHub Releases](https://github.com/getappmap/appmap-intellij-plugin/releases)
-- Download the plugin as a `zip` file.
-- Open IntelliJ and [install the plugin via the `zip` file](https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk).
-- Verify everything is working as intended
-    - AppMap rendering
-    - Installation instructions
-    - Indexing and scanning
-    - Authentication
-- Navigate to the [AppMap plugin page](https://plugins.jetbrains.com/plugin/16701-appmap).
-- As an authorized user, click `Upload Update`.
-- Select the officially built plugin zip and upload to the `Stable` channel.
-
-## Screenshot
-
-[Open screenshot](./appmap-intellij.png)
-
-<img alt="Screenshot of AppMap with IntelliJ" src="appmap-intellij.png" width="100%">
+AppMap is a developer tools platform powered by runtime analysis and AI. It's composed of three components:
+
+* Navie - AI code architect.
+* AppMap data - Runtime traces of your application runtime.
+* AppMap diagrams - Interactive visualizations of AppMap data.
+
+By using AppMap data, Navie is the first AI code architect with the context to understand how your app works when it runs, like a principal engineer or software architect. AppMap data provides Navie with accurate information about application behavior, APIs, database queries, and more, providing valuable context so you can ask more challenging questions and get better answers.
+
+![image](https://github.com/getappmap/appmap-intellij-plugin/assets/1229326/ed36497e-d21d-4b79-be18-5fe4756f4649)
+
+## Get started
+1. **Install [the AppMap Plugin](https://plugins.jetbrains.com/plugin/16701-appmap)** from within the code editor or from the marketplace.  
+
+2. **Sign in with an email address, or with GitHub or GitLab** and Navie will be available in `@explain` mode. This enables Navie to respond to general coding and development questions and answer questions about using AppMap data.
+
+3. **Ask Navie** for guidance recording AppMap data specific to interactions or code scenarios you're interested in analyzing.
+
+   
+## Examples
+Here are some examples of Navie making context-aware suggestions, providing tactical solutions, and reasoning about the larger context of the specific code being worked on.
+
```

Comment:
Looks good overall, we just have to find a way to get an IntelliJ example in here - that's our biggest segment.

Reply:
Java example added.

Reply:
If you wanted to change the description, which appears on the JetBrains Marketplace, then https://github.com/getappmap/appmap-intellij-plugin/blob/develop/description.md would be the file to change.

Reply:
Thanks @jansorg I updated `description.md`